### PR TITLE
Ansible playbook for branding console

### PR DESCRIPTION
**Pull request by Jooho**
*PR Description:*
Ansible playboks are for [branding console](http://playbooks-rhtconsulting.rhcloud.com/playbooks/operationalizing/branding_console.html): changing logo of openshift web console, generating new login page with a new logo.

*Contents:*
This PR contains 2 ansible playbooks :
- ansible-playbook-openshift-custom-login-page
- ansible-playbook-openshift-custom-webconsole-logo

The roles that the 2 playbooks used are in ansible-galaxy:
- [Jooho.image-resize](https://galaxy.ansible.com/Jooho/image-resize/)
- [Jooho.openshift-custom-webconsole-logo](https://galaxy.ansible.com/Jooho/openshift-custom-webconsole-logo/)
- [Jooho.openshift-custom-login-page](https://galaxy.ansible.com/Jooho/openshift-custom-login-page/)

### CODE REVIEW

**Comment by etsauer** [comment]
@redhat-cop/cant-contain-this can someone review?

**Comment by sabre1041** [comment]
@Jooho I had several issues executing and validating the playbook

* When using the _copy_ module where the source is also on the remote location, the `remote_src=true` must be specified
* Numerous warnings  related to _when_ statements being wrapped by `{{ }}`
* Use of curl command instead of [get_url](http://docs.ansible.com/ansible/get_url_module.html) module

The following errors were experienced when attempting to restart OpenShift

```
Jun 29 23:41:40 master-0.sample.casl.example.com atomic-openshift-master[94598]: Invalid MasterConfig /etc/origin/master/master-config.yaml
Jun 29 23:41:40 master-0.sample.casl.example.com atomic-openshift-master[94598]: oauthConfig.templates.login: Invalid value: "/etc/origin/master/stylesheet/images/new-login.html": template is missing parameter {{ .Error }}
Jun 29 23:41:40 master-0.sample.casl.example.com atomic-openshift-master[94598]: oauthConfig.templates.login: Invalid value: "/etc/origin/master/stylesheet/images/new-login.html": template is missing parameter {{ .Names.CSRF }}
Jun 29 23:41:40 master-0.sample.casl.example.com atomic-openshift-master[94598]: oauthConfig.templates.login: Invalid value: "/etc/origin/master/stylesheet/images/new-login.html": template is missing parameter {{ .Names.Password }}
Jun 29 23:41:40 master-0.sample.casl.example.com atomic-openshift-master[94598]: oauthConfig.templates.login: Invalid value: "/etc/origin/master/stylesheet/images/new-login.html": template is missing parameter {{ .Values.Then }}
Jun 29 23:41:40 master-0.sample.casl.example.com atomic-openshift-master[94598]: oauthConfig.templates.login: Invalid value: "/etc/origin/master/stylesheet/images/new-login.html": template is missing parameter {{ .Values.CSRF }}
Jun 29 23:41:40 master-0.sample.casl.example.com atomic-openshift-master[94598]: oauthConfig.templates.login: Invalid value: "/etc/origin/master/stylesheet/images/new-login.html": template is missing parameter {{ .Values.Username }}
Jun 29 23:41:40 master-0.sample.casl.example.com atomic-openshift-master[94598]: oauthConfig.templates.login: Invalid value: "/etc/origin/master/stylesheet/images/new-login.html": template is missing parameter {{ .Action }}
Jun 29 23:41:40 master-0.sample.casl.example.com atomic-openshift-master[94598]: oauthConfig.templates.login: Invalid value: "/etc/origin/master/stylesheet/images/new-login.html": template is missing parameter {{ .Names.Then }}
Jun 29 23:41:40 master-0.sample.casl.example.com atomic-openshift-master[94598]: oauthConfig.templates.login: Invalid value: "/etc/origin/master/stylesheet/images/new-login.html": template is missing parameter {{ .Names.Username }}
```

**Comment by Jooho** [comment]
@sabre1041 oh this ansible was designed to execute from master1 so it didn't need remote_url. However, I think I need to update the script to be executable from anywhere.

I will do it as soon as possible.

Thanks,
Jooho Lee.

**Comment by Jooho** [comment]
@sabre1041 

I update scripts. Please test it again after update ansible-galaxy roles : ansible-galaxy install -r requirements.yaml --force 

Thanks!

**Comment by etsauer** [comment]
@Jooho can you rebase your branch?

File: ansible-playbook-openshift-custom-webconsole-logo/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+- Use your own logo
+~~~
+ansible-playbook ./playbook.yaml -e input_img=/path/to/logo.png
+~~~
+
+After Work
+----------
+~~~
+# Restart Openshift Master Server restart
```
Comment (sabre1041) [inline ansible-playbook-openshift-custom-webconsole-logo/README.md]: Can we add logic to avoid this step?

**Comment by Jooho** [comment]
@sabre1041 Thanks for your review. Actually it was supposed to execute by root user but now I changed it. So now non privileged user can execute it but still the non privileged user need to be added to sudoers file. Moreover, I add the logic that restart master nodes but sometimes it needs to restart masters manually so I add environment `restart_master=false`  

Lastly, I add video clip for better understanding how it works. Please review it again.
Thanks,
Jooho Lee.

**Comment by sabre1041** [comment]
@Jooho Awesome job. Thanks for working through all the requests and changes. Love the videos too. Very useful for showcasing how to walk through each item.

**Pull request merged**